data-reconciliation-app/src/app.json

```diff
@@ -1,21 +1,53 @@
 {
   "endpoints": {
-    "/votes": {
-      "get": {
-        "js_module": "endpoints/sample.js",
-        "js_function": "read",
-        "forwarding_required": "sometimes",
-        "authn_policies": [],
-        "mode": "readonly",
-        "openapi": {}
-      },
+    "/ingest": {
       "post": {
-        "js_module": "endpoints/sample.js",
-        "js_function": "write",
-        "forwarding_required": "sometimes",
-        "authn_policies": [],
+        "js_module": "endpoints/ingest.js",
+        "js_function": "submitData",
```

Comment:
Everything this endpoint receives data. Removing it from the function name could look cleaner.
Given the endpoint is REST-ish and it is called `/ingest`, the method can be `postHandler`?

This could read as: `HTTP POST handler of ingest endpoint`?